### DISCUSSION TITLE
Huge Aggregated EVTX  "WEC"

### DISCUSSION THREAD

**Discussion by Skyw3lker** [Yamato-Security/WELA-deprecated#93] [Q&A]
I'm trying to get an analysis of Event IDs coming from an aggregated EVTX "Forwarded Events" from WEC server ~30 GB.

![wela](https://user-images.githubusercontent.com/7090718/205302739-bd431008-dfc1-4728-a715-c8f7b706423d.jpg)

Is this is a valid use case for the tool ?!

**Comment by hitenkoku** [comment]
@Skyw3lker Thanks for the comment.
We treat it as Int32 by default for large data sizes.

For analysis of evtx files of this size, I recommend using hayabusa(https://github.com/Yamato-Security/hayabusa), 
but I will look into whether WELA can also be run on large data without error.

**Reply by hitenkoku** [reply]
@Skyw3lker  Fixed an error in parsing file size when parsing huge data and merged main branch. Can you check if the problem recurs?

https://github.com/Yamato-Security/WELA/commit/3a69c2d0fb825eb572dcf5e5416610083c235243

**Reply by Skyw3lker** [reply]
@hitenkoku  Thank you so much for recommending the AMAZING hayabusa. I made a git pull, still same exact error.

**Reply by hitenkoku** [reply]
Thank you for your comment.
I will try to find a large amount of data on my end and see if the same problem occurs.
